jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -38,7 +40,7 @@
 import static org.eclipse.jkube.kit.common.util.EnvUtil.storeTimestamp;
 import static org.junit.Assume.assumeFalse;
 import static org.junit.Assume.assumeTrue;
-
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Why is this necessary?

https://github.com/eclipse/jkube/pull/1609#discussion_r897830489